sources/platform/monitoring/index.md

```diff
@@ -41,12 +41,22 @@ Currently, the monitoring option offers the following features:
 
 ### Alert configuration
 
-When you set up an alert, you have two choices for how you want the metrics to be evaluated. And depending on your choices, the alerting system will behave differently:
+When you set up an alert, you have four choices for how you want the metrics to be evaluated. And depending on your choices, the alerting system will behave differently:
```

Comment:
as a part of the validation, new monitoring possibilities arised, I have added them here